source/manual/auto-scaling-groups.html.md

```diff
@@ -0,0 +1,65 @@
+---
+owner_slack: "#re-govuk"
+title: Auto Scaling Groups
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-09-11
+review_in: 6 months
+---
+
+In AWS, [we use auto scaling groups][asg] to ensure we have the right number of
+machines available to handle incoming traffic.
+
+Unfortunately there are a number of limitations with the system currently which
+means we can't use it to automatically scale up the number of machines.
+However, we can manually scale up the number of machines in advance if we
+anticipate an increase in traffic. This was used effectively after we deployed
+the ["Get ready for Brexit" tool][brexit-tool] to ensure we would cope with
+the load.
+
+[asg]: https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html
+[brexit-tool]: https://www.gov.uk/get-ready-brexit-check
+
+## Manually scaling up/down
+
+1. [Access the AWS Console][aws-console] and [go to the EC2 service][ec2-home].
+
+1. Select "Auto Scaling Groups" from the bottom of the menu on the left hand
+   side and find the right machine class in the list (you can filter on the
+   name).
+
+   ![Filtering auto-scaling groups](images/auto-scaling-groups-filter.png)
+
+1. In the "Details" tab at the bottom, you will see "Desired Capacity", "Min"
+   and "Max" which shows the existing configuration. Scroll right and then
+   click on the "Edit" button.
+
+1. In the box that appears, change the numbers as required. To ensure you get
+   the right number of machines you want, it's best to change all three numbers
+   to the same value.
+
+   ![Editing auto-scaling groups](images/auto-scaling-groups-edit.png)
+
+1. Click "Save". This should trigger the creation of new machines and
+   automatically run the appropriate [`Deploy_Node_Apps`][deploy-node-apps]
+   jobs.
+
+1. To check the machines are recognised, you can use
+   `govuk_node_list -c <class>` on the jumpbox and check the IP addresses
```

Comment:
Do you think it would be worth adding a link to jumpbox instructions here @thomasleese ?

Reply:
Good idea, added that to https://github.com/alphagov/govuk-developer-docs/pull/1987.